lib/server/db/seeds/connection.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Utility for adding configurable Connections seeds to the
+ * database.
+ *
+ * @module ConnectionSeed
+ */
+var User = require("../model/User");
+var Connection = require("../model/Connection");
+
+/**
+ * @typedef {Object} ConnectionSeedConfig
```

Comment:
Typedefs allow us to name and explain in detail what ad-hoc objects consist of. They can be used as types in other pieces of JSDoc to give more meaning to our functions.